src/avatar.scss

```diff
@@ -62,6 +62,11 @@ $block: #{$fd-namespace}-avatar;
         bottom: map_get($size-set, "offset");
         right: map_get($size-set, "offset");
 
+        @include fd-rtl() {
+          right: auto;
+          left: map_get($size-set, "offset");
```

Comment:
might want to use this as a variable

Reply:
no need to be a variable, it's using map_get